pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtEngine.java

```diff
@@ -37,6 +37,7 @@
     private static final Logger logger = LoggerFactory.getLogger(PtEngine.class);
 
     public static final String ENGINE_NAME = "PyTorch";
+    static final int RANK = 2;
```

Comment:
Is this something we decide? Why do we have to provide this real rank? 
User can specify their choice of engine and that becomes RANK 1 right ?

Reply:
The rank here is just a way to provide a deterministic way for the default engine.
1. User can always use -Dai.djl.default_engine to choose the default engine.
2. All the engine are available as long as they are included in class path